src/status_im/ui/screens/communities/community_overview_redesign.cljs

```diff
@@ -48,121 +50,217 @@
+          :right-section-buttons [{:icon :main-icons2/search
+                                   :background-color (icon-color)}
+                                  {:icon :main-icons2/options
+                                   :background-color (icon-color)
+                                   :on-press #(>evt [:bottom-sheet/show-sheet
+                                                     {:content (constantly [options-menu/options-menu community])
+                                                      :content-height 400}])}]
+          :left-section {:icon                  :main-icons2/close
+                         :icon-background-color (icon-color)
+                         :on-press #(>evt [:navigate-back])}}]
+        (when (>= @scroll-height @first-channel-height)
+          [react/blur-view {:blur-amount 32
+                            :blur-type :xlight
+                            :overlay-color colors/white-opa-70
+                            :style {:position :absolute
+                                    :top  48
+                                    :height 34
+                                    :width "100%"
+                                    :flex 1}}
+
+           [divider-label/divider-label
+            {:label (:label (last (filter (fn [channel-height]
+                                            (>= @scroll-height (+ (:height channel-height) @first-channel-height)))
+                                          @channel-heights)))
+             :chevron-position :left}]])]
+       [rn/scroll-view {:position :absolute
+                        :top -48
+                        :shows-vertical-scroll-indicator false
+                        :scroll-event-throttle 1
+                        :overflow :scroll
+                        :style {:border-radius (diff-with-max-min @scroll-height 16 0)
+                                :height "100%"}
+                        :on-scroll #(swap! scroll-height (fn [] (-> % .-nativeEvent.contentOffset.y)))}
+
+        [rn/view {:style {:height 151}}
+         [rn/image
+          {:source      cover
+           :style  {:overflow :visible
+                    :flex 1}}]]
+        [rn/view {:flex 1
+                  :border-radius (diff-with-max-min @scroll-height 16 0)
+                  :background-color (colors/theme-colors
+                                     colors/white
+                                     colors/neutral-90)}
+         [rn/view
+          [rn/view {:padding-horizontal 20}
+           [rn/view {:border-radius    40
+                     :border-width     1
+                     :border-color     colors/white
+                     :position         :absolute
+                     :top              (if (<= @scroll-height scroll-0)
+                                         -40
+                                         (min (+ -40 (*  (+ scroll0 @scroll-height) 3)) 8))
```

Comment:
This is going be really easier with thread first or thread max .. `->` `->>`

Reply:
ok cool, I'll take a look to refactor it :)